doc/datasetManager.md

```diff
@@ -274,6 +274,131 @@ webApp:
 
 </details>
 
+### Category details
+
+In category accordions, you can open a category details dialog that can display into a Table component, some twingraph dataset data.
+These data are fetched from a Cypher request sent to the twingraph dataset.
+The table is an expression of the twingraph dataset
+The table is read only.
+The category details dialog can be defined by the key
+`[workspace].webApp.options.datasetManager.categories.previewTable`.
+The value for this field must be an **objects** that contains attributes with the
+following fields:
```

Comment:
typo: _object_ (sg).  Also, I think it's to to talk about `keys` and `values` when we define an object :slightly_smiling_face: